ledger-tool/src/main.rs

```diff
@@ -1,12 +1,11 @@
 #![allow(clippy::integer_arithmetic)]
 use {
-    crate::{bigtable::*, ledger_path::*, output::*},
+    crate::{bigtable::*, ledger_path::*, output::*, utils::*},
```

Comment:
How about `blockstore.rs`? Does this work?

Reply:
Maybe `open_blockstore.rs` to disambiguate from `solana_ledger::blockstore`?

Reply:
There is specifically non-blockstore stuff getting moved, such as the CLI parsing function `get_accounts_db_config()`

Reply:
> There is specifically non-blockstore stuff getting moved, such as the CLI parsing function `get_accounts_db_config()`

Yeah, maybe accountsDb-related stuff should be in a separate module.

Reply:
ok to move those functions to  accounts_db.rs then?

Reply:
yes, that was my reason for putting them all in utils.rs originally. How about for now, I rename `utils.rs` to `ledger_utils.rs` and then in the future when more refactoring is done I'll split in smaller modules?   My main goal is to get the new command into ledger-tool at the moment, see #31346 if interested.

Reply:
> There is another function, `hardforks_of()`, that parses `matches: &ArgMatches<'_>`. Thoughts on putting this one and `get_accounts_db_config` in something like `cli_utils`?

This wfm. It would actually be nice to break out the ledger-tool cli and arg parsing entirely from the command processing one of these days (we have precedent for `cli.rs` and `args.rs/arg_parser.rs`)

> Also, `load_bank_forks()` operates on both blockstore and accounts db, and is beyond just opening blockstore. I know @CriesofCarrots mentioned not wanting very generic module name `utils`, but would you be happy with `ledger_utils` ?

I don't love it, but haven't thought of anything better, assuming you're not game for a separate `load_bank_forks.rs`

Reply:
> This wfm. It would actually be nice to break out the ledger-tool cli and arg parsing entirely from the command processing one of these days (we have precedent for cli.rs and args.rs/arg_parser.rs)

Yeah, I was actually thinking the same thing. I have a plane ride today, but I can take a swing at it then. Definitely no need for @dmakarov to bite that off in this PR IMO

> I don't love it, but haven't thought of anything better, assuming you're not game for a separate load_bank_forks.rs

That's fair. As I see it, the groupings of function we have are:
- 2 arg parser functions: `hardforks_of()` and `get_accounts_db_config()`
- 3 blockstore related functions: `get_shred_storage_type()`, `open_blockstore()`, `open_blockstore_with_temporary_primary_access()`
- `load_bank_forks()`
- `open_genesis_config_by()`

I think the arg parsing functions in one module for this PR make sense, to be later updated by more expansive cli/arg overhaul.

I was then trying to come up with a catch-all for the other 5 functions. I like using specific module names too, but if we get too specific, I feel like new functions may not quite fit and we might end up with a handful of modules with a single function in them (which I guess isn't technically a bad thing)

Reply:
So what do we settle on for this PR?

Reply:
> I think the arg parsing functions in one module for this PR make sense, to be later updated by more expansive cli/arg overhaul.
> 
> I was then trying to come up with a catch-all for the other 5 functions.

I definitely don't think single-function modules are bad :) But since we know we need a lot more reorg in solana-ledger-tool, I'm on board with doing this ^ for now, ie. arg-parsing in one module, other methods in `ledger_utils.rs`. At least ledger_utils tells me that the stuff in it is related to the functionality in `solana-ledger`.

Reply:
I moved arg parsing functions to a new module `args.rs` and renamed the `utils.rs` module to `ledger_utils.rs`.